### PR TITLE
Scan logs for secrets terraform

### PR DESCRIPTION
**Pull request by dekiel**
**Description**

Changes proposed in this pull request:

Terraform configuration deploying components of secret leaks log scanner application. Configuration use google bucket remote state and lock.
Config manage all resources for log scanner.

**ToDo**
- [x] Update config with production settings.

### CODE REVIEW

**Review by KacperMalachowski — COMMENTED** [review]
I have two small suggestions. I put comments at first occurrence of both, but problems are in a few files.

File: configs/terraform/secrets-leaks-log-scanner/gcs-bucket-mover.tf [hunk 1]
```diff
@@ -0,0 +1,76 @@
+resource "google_service_account" "gcs_bucket_mover" {
+  account_id   = "gcs-bucket-mover-cr"
+  description = ""
```
[the remark this answers is not in the document]
Comment (dekiel) [reply configs/terraform/secrets-leaks-log-scanner/gcs-bucket-mover.tf:3]: I forgot to add some description.

File: configs/terraform/secrets-leaks-log-scanner/gcs-bucket-mover.tf [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+    annotations = {
+      "run.googleapis.com/client-name" = "terraform"
+    }
+  }
+
+  template {
+    spec {
+      service_account_name = google_service_account.gcs_bucket_mover.email
+      containers {
+        image = "europe-docker.pkg.dev/kyma-project/dev/test-infra/movegcsbucket:PR-6689"
```
[the remark this answers is not in the document]
Comment (dekiel) [reply configs/terraform/secrets-leaks-log-scanner/gcs-bucket-mover.tf:44]: I think that image tag will not change to often, moreover I expect that image tag will be maintained by automation. For now I will test dependabot support for terraform. If it wont meet requirements, I will check renovate tool. With that the image should be written as full URL to let dependency update tools detect them. For now I will keep this format or image property.

File: configs/terraform/secrets-leaks-log-scanner/gcs-bucket-mover.tf (hunk 1 above)
[the remark this answers is not in the document]
Comment (dekiel) [reply configs/terraform/secrets-leaks-log-scanner/gcs-bucket-mover.tf]: Added descrptions.

**Comment by dekiel** [comment]
/override dekiel_test_of_prowjob_pull-secret-leaks-log-scanner-tf-plan
/override dekiel_test_of_prowjob_pull-build-image-terraform-executor

**Review by KacperMalachowski — APPROVED** [review]

**Pull request merged**